.circleci/config.yml

```diff
@@ -134,7 +134,7 @@ jobs:
                     circleci-agent step halt
                   else
                     source .venv/bin/activate
-                    python -m localstack.testing.testselection.scripts.generate_test_selection_from_pr /tmp/workspace/repo $CI_PULL_REQUEST target/testselection/test-selection.txt
+                    PYTHONPATH=localstack-core python -m localstack.testing.testselection.scripts.generate_test_selection_from_pr /tmp/workspace/repo $CI_PULL_REQUEST target/testselection/test-selection.txt
```

Comment:
This should be removed after https://github.com/localstack/localstack/pull/10830 has been merged, right?

Reply:
no this has nothing to do with the plugin loading, but just the src layout.

without installing localstack with `pip install -e .` into the activated venv, the python binary cannot find the source code when a src layout is used.
another option is to cd into `localstack-core`, or move test selection scripts out of ls all together